todoman/ui.py

```diff
@@ -343,12 +367,27 @@ def compact(self, todo):
             list=todo.list.name,
             percent=todo.percent_complete,
             summary=todo.summary,
-            # XXX: Move this into Todo itself and dedupe it
-            urgent=todo.priority not in [None, 0],
+            priority=todo.priority,
         )
 
         return json.dumps(data, sort_keys=True)
 
+    def parse_priority(self, priority):
+        if priority is None:
+            return 0
+        prio_int = int(priority)
```

Comment:
Concretely, this line will throw an exception whenever `priority` is not an "integer"